tensorboard/plugins/base_plugin.py

```diff
@@ -75,6 +75,18 @@ def is_active(self):
     """
     raise NotImplementedError()
 
+  def es_module_path(self):
+    """Returns one of the keys in get_plugin_apps that is an entry ES module.
+
+    For a plugin that is loaded onto an iframe, a frontend entry point has to be
+    specified. For a plugin that is bundled with TensorBoard as part of
```

Comment:
Okay—we can specify the interface more concretely by pointing to a
frontend TypeScript declaration once we have that part of the system
built. (No action required.)